.gitignore

```diff
@@ -31,3 +36,5 @@ build/
 
 ### VS Code ###
 .vscode/
+
+*application-local.yml
```

Comment:
This allows you to have an application-local.yml file in the resources path that won't be pushed. It will be good for adding stage and production secrets without pushing them. You can set your local environment by setting `-Dspring.profiles.active=local` in the VM options of the application run configurations.